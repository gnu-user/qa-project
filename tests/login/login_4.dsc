TITLE
    Login User Authentication Error
DESCRIPTION
    Attempt to initiate the login transaction by entering “login” into the 
    input, and then read in the current user accounts file. Verify that the 
    output contains a prompt for entry of a username. Enter an invalid username 
    and verify that it is not accepted and that an error occurs.
DEPENDENCIES
    login_3
