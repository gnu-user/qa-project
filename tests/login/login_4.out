Welcome.
> 
Enter username: 
Invalid username.